rest-api-spec/src/main/resources/rest-api-spec/api/indices.get_upgrade.json

```diff
@@ -47,6 +47,10 @@
         "default":"open",
         "description":"Whether to expand wildcard expression to concrete indices that are open, closed or both."
       }
+    },
+    "deprecated":{
```

Comment:
It appears that this field causes all sorts of test failures. I believe only the `"parts"` sub-objects support deprecations in the REST spec for now, so this should be removed and "DEPRECATED" added to the description

Reply:
This is my fault, I used wrong `gradle check` command and missed this tests. So, I've fixed it and some other tests in new commit.

Reply:
I used this doc for deprecation: https://github.com/elastic/elasticsearch/tree/master/rest-api-spec#deprecations. Currenlty it incorrectly shows how to deprecate entire API. Is it a inaccuracy in the documentation or big in a code?

Reply:
hi @dakrone ! Could you please check my last comment above? Should we do smth with that the doc is incorrect?